EventView-1/src/main/java/com/eventview/service/EmailService.java

```diff
@@ -15,23 +17,40 @@
     @Autowired
     private JavaMailSender javaMailSender;
 
-    private EventTypeService eventTypeService;
-    private UserService userService;
 
-    public void sendTextEmail(String sendTo, String subject, String body1, String body2) {
 
-        SimpleMailMessage msg = new SimpleMailMessage();
+    public EmailService() {
+    }
+
+    public void sendTextEmail(String sendTo, List<String> fNamesNext, List<String> eventTypesNext, List<String> fNamesToday, List<String> eventTypesToday) {
```

Comment:
u still doing wrong. don't send emails to users send emails only to admin.

Reply:
yeah I will remove this and take email from properties